conf/modules.config

```diff
@@ -446,6 +446,13 @@ process {
     }
 
     withName: MOTUS_PROFILE {
+        ext.args = {
+            [
+                params.motus_ncbi_id ? "-p" : "",
+                params.motus_read_counts ? "-c" : "",
+                params.motus_mgc_counts ?  "-M ${task.ext.prefix}.mgc" : ""
```

Comment:
Can't this just be `${prefix}`?

Reply:
This is how I understood the implementation from our meeting on Tuesday or what do you think @Midnighter?

Reply:
Hmm, I don't know off the top of my head. It depends a lot on when this gets evaluated. My intuition was that the config is evaluated when `input` is available but before the script block. If that is true then `task.ext.prefix` would be defined but `prefix` not yet.

Reply:
Ah ok. @sofstam maybe you could do a little test and report back? This would be useufl knowledge :)

Reply:
Yes, I can do that :)

Reply:
So, if I write: `${prefix}.mgc`, the output file looks like: `[:].mgc` whereas with `${task.ext.prefix}.mgc`, I get the expected filenames.